src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -48,6 +48,7 @@ public static partial class AppHostBuilderExtensions
 
 		public static IMauiHandlersCollection AddMauiControlsHandlers(this IMauiHandlersCollection handlersCollection)
 		{
+			handlersCollection.AddHandler<VisualElement, DefaultViewHandler>();
```

Comment:
This is the real fix - adding something for the controls to fall back to.

Reply:
I think this will help Third Party Vendors and when it comes to migrating code.

Reply:
So is the idea that we call this in the static constructor of our controls?

Reply:
Actually... on second look, that wouldn't work, since I don't have a reference to the handlersCollection. How does this help 3rd party vendors?

Reply:
@dotMorten Maybe this is something that will interest you: https://github.com/dotnet/maui/tree/main/src/Controls/samples/Controls.Sample/Controls/BordelessEntry

Basically, instead of getting all vendors to register all views in a single place, you can create some shared `appBuilder.UseMyVendorThings()` and then get a static access to the handler collection and then add handlers to it at any time.

The reason we never added the global assembly search as this often caused renderers to either be linked away or not at all. So we maybe went the other extreme and required handlers to always be manually registered in the builder.

However, this is more of a middle ground where each vendor can use a shared assembly to have a single `UseXxx`? However, maybe we do need some static access to this collection in the Controls level to make things easier for everyone? Not sure we are ready to add this in the Core layer, but maybe for Controls we can have a static `DynamicHandlersCollection` or something that can get access to the collection and then anyone can add to there.

But a counter argument to requiring an explicit `UseXxx` is that often if there was no reference to the render dll, MSBuild would not even load it nor will it ever reach an app. So often there would be some dummy `Xxx.Init()` call in some place.

What we have now is similar, but also much more discoverable and user friendly. When a user installs some UI NuGet, then they go the the app builder and preses the `.`

Reply:
OK, so gonna totally re-use this PR for TemplatedView as that needs a handler too - just like ContentView.

Reply:
I already have the single UseXXX thing. It still mean things doesn’t “just” work. And it’s not like we aren’t using these controls. Typically they are referencing in xaml and code behind as well. Now I realize that your current system doesn’t “see” what is in the xaml at compile time, so no reference is generated but I don’t see why that isn’t addressable or why you can’t generate a code snippet at compile time to register these classes.

Reply:
We moved this specific issue to here https://github.com/dotnet/maui/issues/7258

It is something we want to do.

Reply:
Since the Xaml is always compiled, we do have the first part being referenced. The trick is the link between the view and the handler.